src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -271,7 +274,7 @@ private AzureFileSystem GetAzureFileSystem()
         {
             LocalSettings settings = GetSettings();
             AzureFileSystem fileSystem = GetAzureFileSystem();
-            bool result = await InitCommand.RunAsync(settings, fileSystem, true, true, new SleetLogger(Log), CancellationToken);
+            bool result = await InitCommand.RunAsync(settings, fileSystem, true, false, new SleetLogger(Log), CancellationToken);
```

Comment:
What is this bool change?

Reply:
It was my test yesterday of push/not push symbols. I added that in my change but somehow the commit is not here